docs/source/en/perf_train_gpu_one.mdx

```diff
@@ -9,489 +9,257 @@ Unless required by applicable law or agreed to in writing, software distributed
 
-    Linear layers and components of Multi-Head Attention all do batched **matrix-matrix multiplications**. These operations are the most compute-intensive part of training a transformer.
+Note: a bracket means there may be negligible effects on memory utilization. 
 
-2. **Statistical Normalizations**
+You can combine the above methods to get a cumulative effect. These techniques are available to you whether you are 
+training your model with [`Trainer`] or writing a pure PyTorch loop, in which case you can [configure these optimizations 
+with 🤗 Accelerate](#using--accelerate).
 
-    Softmax and layer normalization are less compute-intensive than tensor contractions, and involve one or more **reduction operations**, the result of which is then applied via a map.
+If these methods do not result in sufficient gains, you can explore the following options: 
+* [Look into building your own custom Docker container with efficient softare prebuilds](#efficient-software-prebuilds)
+* [Consider a model that uses Mixture of Experts (MoE)](#mixture-of-experts)
+* [Convert your model to BetterTransformer to leverage PyTorch native attention](#using-pytorch-native-attention)
 
-3. **Element-wise Operators**
+Finally, if all of the above is still not enough, even after switching to a server-grade GPU like A100, consider moving 
+to a multi-GPU setup. All these approaches are still valid in a multi-GPU setup, plus you can leverage additional parallelism 
+techniques outlined in the [multi-GPU section](perf_train_gpu_many). 
 
-    These are the remaining operators: **biases, dropout, activations, and residual connections**. These are the least compute-intensive operations.
+## Batch size choice
 
-This knowledge can be helpful to know when analyzing performance bottlenecks.
+To achieve optimal performance, start by identifying the appropriate batch size. It is recommended to use batch sizes and 
+input/output neuron counts that are of size 2^N. Often it's a multiple of 8, but it can be 
+higher depending on the hardware being used and the model's dtype.
 
-This summary is derived from [Data Movement Is All You Need: A Case Study on Optimizing Transformers 2020](https://arxiv.org/abs/2007.00072)
+For reference, check out NVIDIA's recommendation for [input/output neuron counts](
+https://docs.nvidia.com/deeplearning/performance/dl-performance-fully-connected/index.html#input-features) and 
+[batch size](https://docs.nvidia.com/deeplearning/performance/dl-performance-fully-connected/index.html#batch-size) for 
+fully connected layers (which are involved in GEMMs (General Matrix Multiplications)).
 
+[Tensor Core Requirements](https://docs.nvidia.com/deeplearning/performance/dl-performance-matrix-multiplication/index.html#requirements-tc) 
+define the multiplier based on the dtype and the hardware. For instance, for fp16 data type a multiple of 8 is recommended, unless 
+it's an A100 GPU, in which case use multiples of 64.
 
-## Anatomy of Model's Memory
```

Comment:
I don't think it's a good idea to remove completely this aprt of the guide.

Reply:
@sgugger The main feedback to these parts of the documentation was that it was difficult to find actionable information. It was a large document that was hard to navigate (it is still large, but in my opinion, more actionable).  Less doc is precisely what users wanted. Perhaps, the anatomy of the model can move to the conceptual guides, and we can link it here. As for the benchmarking, here's my reasoning for removing the examples - a) the benchmarks will differ depending on a number of factors, and one example does not guarantee the same results for all scenarios, b) there are links to benchmarks at the end of some sections. Perhaps, instead of this exact example throughout all the doc, we could add a separate doc about benchmarking (how to) so that users could do that on their own. 

Thank you for adding @lvwerra and @stas00, would love to hear their thoughts!